doc/ref/language.xml

```diff
@@ -1804,12 +1800,16 @@ The user should compare the above with the  &GAP; function <Ref Func="Sum"/>
 which, for example, may take a list argument and optionally
 an initial element (which zero should the sum of an empty list return?).
 <P/>
-Note that if a function <A>f</A> is defined as above with its last formal
-argument <C>arg</C> then <C>NumberArgumentsFunction(<A>f</A>)</C> returns
-minus the number of formal arguments (including the <C>arg</C>
+GAP will also treat a function with a single argument with the name <C>arg</C>
+as function with a variable length list of arguments, as if the user had
+written <C>arg...</C>.
+<P/>
+Note that if a function <A>f</A> is defined as above
+then <C>NumberArgumentsFunction(<A>f</A>)</C> returns
+minus the number of formal arguments (including the final argument)
 (see&nbsp;<Ref Func="NumberArgumentsFunction"/>).
 <P/>
-The argument <C>arg</C> when used as the single argument name of some function
+The <C>...</C> notation when used as the single argument name of some function
 <A>f</A> tells &GAP; that when it encounters <A>f</A> that it should form  a  list
 out of the arguments of <A>f</A>.
```

Comment:
That sentence is weird in the revised form... It doesn't really make sense to only talk about the "single argument" case here, right?